### ISSUE TITLE
remove `.localized` from Donate Banner definitions

### ISSUE THREAD

**Issue by danielfmiranda** [MozillaFoundation/foundation.mozilla.org#11404]
As mentioned [here](https://github.com/MozillaFoundation/foundation.mozilla.org/pull/11399/files#r1388532201), we should try to remove the use of .localized when defining the donate banner in templates

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/TP1-202)

**Comment by data-sync-user** [comment]
➤ diego commented:

Hi Simon Acosta Torres,

Taking a look at this ticket I think we can abandon it because the file network-api/networkapi/templates/donate/pages/base.html does not have the .localized attribute anymore. Also digging deeper into related banner templates I found that in bannered_campaign_page.html there was still some .localize pending to be removed but then I realized Daniel Miranda had worked on that already and the PR was ready.

Thank you!
